src/VisualStudio/Core/Def/Implementation/Venus/ContainedLanguage.cs

```diff
@@ -113,6 +114,7 @@ public static string GetFilePathFromHierarchyAndItemId(IVsHierarchy hierarchy, u
             ProjectId projectId,
             VisualStudioProject? project,
             string filePath,
+            string language,
```

Comment:
Check that this doesn't break Typescript if they've moved to the newer constructor.